index.bs

```diff
@@ -22,7 +23,7 @@ img { max-width: 100%; }
 <div boilerplate="status">
 This is an unofficial draft. It is provided for discussion only and may change at any moment. Its publication here does not imply endorsement of its contents by W3C. Don’t cite this document other than as work in progress.
 
-If you wish to make comments regarding this document, please send them to <a href="mailto:public-respimg@w3.org">public-respimg@w3.org</a> (<a href="mailto:public-respimg-request@w3.org?subject=subscribe">subscribe</a>, <a href="http://lists.w3.org/Archives/Public/public-respimg/">archives</a>). All comments are welcome.
+If you wish to make comments regarding this document, please send them to <a href="mailto:public-wicg@w3.org">public-wicg@w3.org</a> (<a href="mailto:public-wicg-request@w3.org?subject=subscribe">subscribe</a>, <a href="https://lists.w3.org/Archives/Public/public-wicg/">archives</a>). All comments are welcome.
```

Comment:
I don't think we want people using the mainline list, right? Comments should only be made on github. Let's delete this.

Reply:
I've remove this paragraph entirely